static/js/stats.js

```diff
@@ -47,7 +47,7 @@ function countMarkers(map) { // eslint-disable-line no-unused-vars
                 pokeCounts.push(
                     [
                         '<img src=\'static/icons/' + pkmnCount[i].ID + '.png\' />',
-                        '<a href=\'http://www.pokemon.com/us/pokedex/' + pkmnCount[i].ID + '\' target=\'_blank\' title=\'View in Pokédex\' style=\'color: black;\'>' + pkmnCount[i].Name + '</a>',
+                        '<a href=\'https://pokemon.gameinfo.io/en/pokemon/' + pkmnCount[i].ID + '\' target=\'_blank\' title=\'View in Pokédex\' style=\'color: black;\'>' + pkmnCount[i].Name + '</a>',
```

Comment:
I would advice not to support locale in the link. Indeed it would be better for user experience, but if it is changed to a php file Grunt cannot process it anymore and make a minified version of it. So more data usage for a small improvement that I don't think many people use.

This is just my opinion.